integration/runtime_test.go

```diff
@@ -157,9 +158,9 @@ func spawnGlobalDaemon() {
 			Scheme: testDaemonProto,
 			Host:   testDaemonAddr,
 		}
-		job := eng.Job("serveapi", listenURL.String())
-		job.SetenvBool("Logging", true)
-		if err := job.Run(); err != nil {
+
+		serverConfig := &apiserver.ServerConfig{Logging: true}
```

Comment:
I dunno why it was `true`, but should be really `false`. And below too. I hope we will kill `integration` suite soon :/